modules/ldap/README.md

```diff
@@ -0,0 +1,87 @@
+# Example to provision LDAP Terraform Module
```

Comment:
This README is the one that belongs in the `example`.  See https://github.com/ibm-hcbt/terraform-ibm-cloud-pak/blob/main/modules/iaf/README.md as an example for the module README